data/changelog/omp/2023-10-23-omp-deprecation.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: OMP deprecation
+title: OMP Deprecation
 date: "2023-10-23"
 tags: [omp, platform, deprecation]
 ---
 
 After [giving space for feedback and objections by the community](https://discuss.ocaml.org/t/rfc-deprecating-ocaml-migrate-parsetree-in-favor-of-ppxlib-also-as-a-platform-tool/13240),
-we have deprecated ocaml-migrate-parsetree (aka OMP). It is superseded by [Ppxlib](https://github.com/ocaml-ppx/ppxlib).
+we have deprecated `ocaml-migrate-parsetree` (aka OMP). It is superseded by [Ppxlib](https://github.com/ocaml-ppx/ppxlib).
 
 There are four major differences between OMP and Ppxlib, which all go hand in hand.
 
-The first major difference is in the library and therefore impacts how to write PPXs:
-With OMP, each PPX author had to choose a parsetree version to define their PPX against.
+The first major difference is in the library and therefore impacts how to write PPXs.
+With OMP, each PPX author had to choose a parsetree version to define against their PPX.
```

Comment:
Thanks for the review, @christinerose! Here's one thing that I think was correct before and is wrong now:

It's actually the other way around: The PPX is defined the parsetree version, not the parsetree version against the PPX.

Reply:
Thanks for letting me know! I wondered about that. I should've checked with you first.

Reply:
Reverted!